input/HSCDR1wl_0.yml

```diff
@@ -0,0 +1,20 @@
+depth_cut: 24.5
+z_edges: [0.3, 0.6]
```

Comment:
By construction? In this case there's really no need. For DESY1 this is necessary because the bin assignment is not as simple, but in this case it is actually as simple as binning in photo-z.